### PR TITLE
Revert dec0e7311d3b4dc5211203c55c1d1ba72e65a84c typeset updates

### PR DESCRIPTION
**Pull request by wguevara-norton**

### CODE REVIEW

File: website/sidebars.js [hunk 1]
```diff
@@ -15,7 +15,6 @@ module.exports = {
 				// 'migration',
 			],
 		},
-		'guides/usable-writing-guidelines',
```
Comment (JRebella) [inline website/sidebars.js:18]: Is this change intended?

**Review by JRebella — APPROVED** [review]
Approved but check the comment on sidebars.js please

**Review by sh0ji — CHANGES_REQUESTED** [review]
This should only have changes to `packages/core/src/type/index.scss`. Just as website changes shouldn't change design system packages, package changes shouldn't change the website.

File: website/src/scss/main.scss [hunk 2]
```diff
@@ -139,8 +139,3 @@ ol ol,
 ul ol {
 	list-style-type: lower-alpha;
 }
-
-h1,
```
[the remark this answers is not in the document]
Comment (wguevara-norton) [reply website/src/scss/main.scss:143]: Evan that change is not from this commit to be reverted

[the remark this answers is not in the document]
Comment (sh0ji) [reply website/src/scss/main.scss:143]: That's fine but I still don't know what that code was trying to accomplish or why it was part of this. I wasn't necessarily asking you to change anything&mdash;I just wanted to understand the change.

**Review by sh0ji — COMMENTED** [review]
We didn't get to go over this today, but I was hoping to ask if this was going to cause the documentation spacing to regress. If our documentation expects bigger heading margins, that should probably be moved to [website/src/scss/main.scss](https://github.com/wwnorton/design-system/blob/main/website/src/scss/main.scss) to avoid this breaking anything.

**Review by sh0ji — APPROVED** [review]

**Comment by sh0ji** [comment]
LGTM! Once I've merged this, can you merge `main` into `modal-broken-spacing` so that it can be properly tested before we merge #258?

**Pull request merged**